Makefile

```diff
@@ -166,116 +166,47 @@ proto-vendor-grpc-health: ## Copy the gRPC Health interface from upstream
+$(1)-test-all: musl $(GEN_RS) $(GEN_TS)
+	sudo -E $(cargo) test --target $(uname_m)-unknown-linux-musl -p $(1) -- --include-ignored
 
-.PHONY: auraed-test-watch
-auraed-test-watch: musl $(GEN_RS) $(GEN_TS) # Use cargo-watch to continuously run a test (e.g. make auraed-test-watch name=path::to::test)
-	sudo -E $(cargo) watch -- $(cargo) test --target $(uname_m)-unknown-linux-musl -p auraed $(name) -- --include-ignored
+$(1)-test-watch: musl $(GEN_RS) $(GEN_TS) # Use cargo-watch to continuously run a test (e.g. make $(1)-test-watch name=path::to::test)
+	sudo -E $(cargo) watch -- $(cargo) test --target $(uname_m)-unknown-linux-musl -p $(1) $(name) -- --include-ignored
 
-.PHONY: auraed-build
-auraed-build: musl $(GEN_RS) $(GEN_TS)
-	$(cargo) build --target $(uname_m)-unknown-linux-musl -p auraed
+$(1)-build: musl $(GEN_RS) $(GEN_TS)
+	$(cargo) build --target $(uname_m)-unknown-linux-musl -p $(1)
 
-.PHONY: auraed-build-release
-auraed-build-release: musl $(GEN_RS) $(GEN_TS)
-	$(cargo) build --target $(uname_m)-unknown-linux-musl -p auraed --release
+$(1)-build-release: musl $(GEN_RS) $(GEN_TS)
+	$(cargo) build --target $(uname_m)-unknown-linux-musl -p $(1) --release
 
-.PHONY: auraed-debug
-auraed-debug: musl $(GEN_RS) $(GEN_TS) auraed-lint
-	$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./auraed --debug --force
+$(1)-debug: musl $(GEN_RS) $(GEN_TS) $(1)-lint
+	$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./$(1) --debug --force
 
-.PHONY: auraed-release
-auraed-release: musl $(GEN_RS) $(GEN_TS) auraed-lint auraed-test ## Lint, test, and install auraed
-	$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./auraed --force
+$(1)-release: musl $(GEN_RS) $(GEN_TS) $(1)-lint $(1)-test ## Lint, test, and install $(1)
+	$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./$(1) --force
 
-.PHONY: start
-auraed-start: ## Starts the installed auraed executable
-	sudo -E $(HOME)/.cargo/bin/auraed
-
-#------------------------------------------------------------------------------#
-
-# AuraeScript Commands
-
-.PHONY: auraescript
-auraescript: $(GEN_TS) $(GEN_RS) auraescript-lint auraescript-debug ## Lint and install auraescript (for use during development)
-
-.PHONY: auraescript-lint
-auraescript-lint: $(GEN_TS) $(GEN_RS)
-	$(cargo) clippy -p auraescript --all-features -- -D clippy::all -D warnings
-
-.PHONY: auraescript-test
-auraescript-test: $(GEN_TS) $(GEN_RS)
-	$(cargo) test -p auraescript
-
-.PHONY: auraescript-test-all
-auraescript-test-all: $(GEN_TS) $(GEN_RS)
-	$(cargo) test -p auraescript -- --include-ignored
-
-.PHONY: auraescript-build
-auraescript-build: musl $(GEN_TS) $(GEN_RS)
-	$(cargo) build -p auraescript
-
-.PHONY: auraescript-build-release
-auraescript-build-release: musl $(GEN_RS) $(GEN_TS)
-	$(cargo) build -p auraescript --release
+.PHONY: $(1) $(1)-lint $(1)-test $(1)-test-all $(1)-test-watch $(1)-build $(1)-build-release $(1)-debug $(1)-release
```

Comment:
Should we keep the `.PHONY`s separate, above each command? I feel like someone (AKA me) will forget to add to this list when we add a command.

Reply:
we can.. i was just trying to save some space.